oeps/best-practices/oep-0067-bp-tools-and-technology.rst

```diff
@@ -0,0 +1,136 @@
+     - 2023-09-05
+   * - Authors
+     - Feanil Patel <feanil@axim.org>
+   * - Arbiter
+     - TBD
+   * - Status
+     - Accepted
+   * - Type
+     - Best Practice
+   * - Created
+     - 2023-09-05
+   * - Review Period
+     - 2023-09-05 - 2023-10-xx
+   * - Resolution
+     - <links to any discussions where the final status was decided>
+   * - References
+     - :doc:`oep-0011-bp-FED-technology`
+
+Abstract
+********
+
+The Open edX Platform is built using many 1st and 3rd party tools and technologies.
+This document covers the current recommendations for the tools and technologies
+that should be used.
+
+Motivation
+**********
+
+The Open edX platform has a lot of 1st and 3rd party tools and technology that it uses.
+The problems we're trying to solve are:
+
+    * It is difficult to know what the current best practice for a given area.
+
+    * It is unclear how to go about the discussion and decision making process
+      to change a best practice.
+
+:doc:`OEP-11 <oep-0011-bp-FED-technology>` serves as a great example for the
+usefulness of this kind of OEP but limits itself to be relevant only to
+frontend technologies.  This OEP would supersede :doc:`OEP-11
+<oep-0011-bp-FED-technology>` and encompass not only frontend technolog but any
+technology that:
+
+    * Is considered a best practice for the Open edX Platform
+
+    * Is relevant to a large subset of the Open edX Platform
+
+Specification
+*************
+
+.. note:: We'll fill this out before the OEP lands but leaving mostly blank for initial feedback.
+
+General Technology Selection
+============================
+
+This technology is not specific to frontend or backend code.
+
+* Codecov is an example in this category.
+
+* Github actions is probably another
+
+Frontend Technology Selection
+=============================
+
+The phrase “frontend” is used to mean any part of the platform that is shown to
+users. This encompasses views rendered in Python on the server, interactive
+interfaces written using JavaScript, and CSS styling.
+
+This will be an exact copy of the content of the **Technology Selection**
+section of OEP-11
+
+Backend Technology Selection
+============================
+
+This section will cover any technology or tools that are not considered a part
+of the frontend.
+
+This section will be empty to begin with and I'll create a separate PR to add any
+technologies that we consider standard today. This will probably include:
+
+    * Django
+
+    * Django Rest Framework
+
+    * Python?
+
+    * drf-yasg (A thing that is currently standard but that I hope to change)
+
+    * tox
+
+    * pytest
+
+    * edx-lint/pylint
+
+
+
+Decisions
+*********
+
+This section lists the decision records for any time a standard technology has
+been change in the system along with the reasons why and the alternatives
+considered.
+
+.. We'll have a toctree to an ADR folder with decisions.  We'll move all the existing fronted decisions to under this OEP.
```

Comment:
Is the idea to have `oeps/best-practices/oep-0067/decisions/thing-we-decided.rst` or `oeps/best-practices/oep-0067/decisions/frontend/thing-we-decided.rst`?

Reply:
I had assumed the first one but I'm very open to the second option.  What's the value of the second option in your mind, I see that it provides a bit more separation but I'm not clear on what that separation buys us.

Reply:
I'm fine either way. I guess I was thinking the separation would buy us a nested TOC. This was asked out of curiosity not opinion.